### PR TITLE
Fix nav width

### PR DESCRIPTION
**Pull request by MMP0**

### CODE REVIEW

**Comment by akuma06** [comment]
Seems like it makes the header taking the full width of the screen

File: public/css/main.css [hunk 1]
```diff
@@ -52,7 +52,7 @@ body {
 
 .header .container {
 	position: relative;
-	width: auto;
+	max-width: none;
```
Comment (akuma06) [inline public/css/main.css]: With this, you remove the max-width of the header. So it takes the whole space and so its width is equal to the width of the screen

**Comment by MMP0** [comment]
Some links disappears again, like this: #915

**Comment by akuma06** [comment]
I know but you people doesn't want a wide header. Might be better to move FAQ link in the footer instead

**Comment by MMP0** [comment]
I see

**Pull request closed without merging**